File: exam921/server-app/src/main/java/org/example/server/domain/Sensor.java
package org.example.server.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import javax.validation.constraints.Min;
import java.io.Serializable;
import java.sql.Time;

@Entity
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Table(name="sensor")
public class Sensor implements Serializable {
  @Id
  @GeneratedValue(strategy=GenerationType.IDENTITY)
  private Integer id;

  private String name;

  private int measurement;

  private Time time;


}

File: exam921/monitor-web/src/main/web-app/src/app/model/sensor.ts
export class Sensor {
  id: number;
  name: string;
  measurement: number;
  time: string

  constructor(id: number, name: string, measurement: number, time: string) {
    this.id = id;
    this.name = name;
    this.measurement = measurement;
    this.time = time;
  }
}
